frontend/src/pages/start/start.tsx

```diff
@@ -108,15 +112,17 @@ const Start: React.FC = () => {
                 <div className={styles['stepper-wrapper']}>
                   <div className={styles['img-wrapper']}>
                     <IconButton
+                      type="button"
                       icon="arrowLeft"
                       label="Go to the previous step"
                       onClick={handlePreviousStep}
                       isDisabled={isFirstStep}
                     />
                     <IconButton
+                      type="submit"
+                      form={STEPS_FORM_SUBMIT}
```

Comment:
change it everywhere
```suggestion
                      formId={STEPS_FORM_SUBMIT_ID}
```